cfgov/unprocessed/css/organisms/form.less

```diff
@@ -0,0 +1,59 @@
+/* topdoc
+  name: Forms
+  family: cf-organisms
+  patterns:
+  - name: Standard form
+    markup: |
+      <form>
+          <h3>Title of form</h3>
+          <p class="o-form-description">Intro text</p>
+          <h3>Description of section</h3>
+          <div class="o-form-section"></div>
+          <div class="o-form-action"></div>
+      </div>
+    codenotes:
+      - |
+        Pattern structure
+        -----------------
+        form
+          h2
+            .o-form-description
+          h3
+            .o-form-section
+          .o-form-actions
+  tags:
+  - cf-organisms
+*/
+
+.o-form-description {
+    margin-top:  unit( @grid_gutter-width / @base-font-size-px, em);
+    margin-bottom:  unit( @grid_gutter-width * 2 / @base-font-size-px, em);
+    .short-desc();
+}
+
+.o-form-section {
+    margin: unit( @grid_gutter-width / @base-font-size-px, em)
+            unit( @grid_gutter-width / -2 / @base-font-size-px, em)
```

Comment:
I think it's easier to read like this since it's only one operation on the grid_gutter-width versus two. I'm not too picky if consistency is more important.